systemtest/src/test/java/io/strimzi/systemtest/ConnectST.java

```diff
@@ -157,6 +161,77 @@ void testKafkaConnectWithFileSinkPlugin() throws Exception {
         KafkaTopicUtils.waitForKafkaTopicDeletion(CONNECT_TOPIC_NAME);
     }
 
+    @Test
+    @Tag(NODEPORT_SUPPORTED)
```

Comment:
what does it mean? sorry for the stupid question :-)

Reply:
The Nodeport supported means that the only environments which support Nodeports for instance in AWS nodeport is working only with specific configuration the same as for load balancers in OCP4. So, with this notation, we are excluding some tests to run only in a specific environment.